canary/apps/react/next/package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@aws-amplify/ui-react": "latest",
+    "@aws-amplify/ui-react": "liveness",
+    "@aws-amplify/ui-react-liveness": "liveness",
```

Comment:
Does this look right? :)

Reply:
I think we can't change this until the package actually releases since it is installing from npm @0618 thoughts?

Reply:
Yes, `liveness` is the version in CA

Reply:
This means that `build-and-runtime` test will fail after this is merged onto `main`. I believe we should skip this canary change until liveness is released, or we can do a tagged release to `@liveness`.